src/blocks/map/edit.js

```diff
@@ -148,6 +148,37 @@ class Edit extends Component {
 			zoom,
 		} = attributes;
 
+		const userLocale = () => {
+			let locale = $( '[class*="locale-"]' ).attr( 'class' ).split( 'locale-' )[ 1 ].split( ' ' )[ 0 ];
+			switch ( locale.toLowerCase() ) {
```

Comment:
https://github.com/godaddy/coblocks/pull/964/files#diff-7ebdffc151d5a58795470ce8ca4ec512R153-R179

So just out of curiosity during testing I wanted to see if there was another way to accomplish this to see if we really needed a switch statement. Turns out we can very easily pull the language attribute from the `<html>` element like this:

```
const locale = document.documentElement.lang;
```

It seems like Google can accept this value without any modifications and load the correct translations. So we can remove this new function entirely and concatenate this value directly here: https://github.com/godaddy/coblocks/pull/964/files#diff-7ebdffc151d5a58795470ce8ca4ec512L243-R274